R/plot_scatter.R

```diff
@@ -211,23 +211,32 @@ plot_scatter <- function(data,
 
 
 check_plot_scatter <- function(data, alignment_limit, cap_outliers, floor_outliers) {
-  abort_if_missing_names(data, c(
+  assert_missing_names(data, c(
```

Comment:
shouldn't the naming be something along the lines of `assert_no_missing_names()`?

Reply:
ah crap, you're right, got sloppy towards the end. That's why I prefer the assertive versions of these names e.g. `assert_*()`... less confusing than the double negatives.

Reply:
done